apps/reward-manager/src/dd_client.ts

```diff
@@ -0,0 +1,74 @@
+const DD_BASE_URL = "https://api.datadoghq.com";
```

Comment:
Same as the notifier.ts file, lets remove this in favor of the existing logging package